_events/sr2019/competition.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Competition
+date: 2019-04-06 9:00:00
+layout: event
+type: competition
+location: Southampton University
+---
+
+## Tickets
+
+Competitors need to bring along a ticket to be allowed entry. Anyone under the
+age of 18 must be accompanied by an adult. Student Robotics will announce when
+tickets are available.
+
+## Directions
+
+* How to [get to campus][soton-campus-directions]
+* [On-campus directions][soton-directions] (and parking)
+
+## More Details
+
+To come in the following weeks.
```

Comment:
This is slightly confusing wording in my opinion. Perhaps "to come in the weeks following the date announcement"